src/components/flame-graph/index.js

```diff
@@ -39,10 +46,25 @@ type StateProps = {|
   +processDetails: string,
   +callNodeInfo: CallNodeInfo,
   +threadIndex: number,
+  +selectedCallNodeIndex: IndexIntoCallNodeTable | null,
+  +profileCallTreeContextMenuVisible: boolean,
```

Comment:
Could you rename `profileCallTreeContextMenuVisible` to `isProfileCallTreeContextMenuVisible`. This is a fairly common naming convention in DevTools.